Dockerfile

```diff
@@ -13,7 +13,7 @@ COPY ocsf-utils/build/libs/ /app/lib
 COPY log4j-core-2.23.1.jar /app/lib
 COPY log4j-api-2.23.1.jar /app/lib
 COPY ocsf-cli.sh /app/lib
-COPY schema.json ocsf-schema/build/schema.json /app/lib
+COPY schema.json /app/lib
```

Comment:
`docker build` seemed to be angry about 3 arguments here. Removing the middle argument seems to result in a smooth build.